### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by kolayne**
Sample Go+HTTP Hello World app

It may look pretty big for a hello world. It's because I've added some simple features to show the basic abilities of the go HTTP framework(?)

### CODE REVIEW

File: src/main.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		dt := time.Now()
+		hour, min, sec := dt.Clock()
+		dtStr := fmt.Sprintf("%d:%d:%d", hour, min, sec)
+
+		if err := tmpl.ExecuteTemplate(w, "/", dtStr); err != nil {
+			panic(err)
+		}
+	})
```
Comment (kolayne) [inline src/main.go]: Should I move this handler to a separate function? If so, how do I name it?

File: src/main.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+	http.HandleFunc("/secret_page", func(w http.ResponseWriter, r *http.Request) {
+		if err := tmpl.ExecuteTemplate(w, "Secret Page", nil); err != nil {
+			panic(err)
+		}
+	})
```
Comment (kolayne) [inline src/main.go]: Should I move this handler to a separate function then? (if there is some kind of a style requirement, this should be moved of course but I find this handler pretty short)

File: src/templates/secret_page.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Secret Page?!</title>
+    </head>
+
+    <body>
+    <h1>OMG!</h1>
+    <p>
+        Have you just visited THE SECRET PAGE?!
```
Comment (kolayne) [inline src/templates/secret_page.html]: Don't place any secret info here! :O

**Review by FixedOctocat — APPROVED** [review]

**Review by FixedOctocat — APPROVED** [review]

File: src/main.go (hunk 1 above)
Comment (DarkSquirrelComes) [reply src/main.go]: Yes, you definitely should. May be "HandleRoot"?

File: src/main.go (hunk 2 above)
Comment (DarkSquirrelComes) [reply src/main.go]: I guess it should be separate. And don't forget to recover panic, pls.

**Review by DarkSquirrelComes — CHANGES_REQUESTED** [review]
Remove anonymous handlers, handle panic.

**Comment by DarkSquirrelComes** [comment]
BTW, you can just log error and return 500 http status instead of panic.

**Review by FixedOctocat — APPROVED** [review]

**Review by tanya-kta — APPROVED** [review]

**Comment by kolayne** [comment]
I don't think we need this anymore. Thanks to everybody

**Pull request closed without merging**